streaming/index.js

```diff
@@ -1177,7 +1177,7 @@ const startServer = async () => {
     const location = url.parse(req.url, true);
 
     req.requestId = uuid.v4();
-    req.remoteAddress = ws._socket.remoteAddress;
+    req.remoteAddress =  req.headers['x-forwarded-for']?.split(',')[0].trim() || ws._socket.remoteAddress;
```

Comment:
Pretty doesn't currently run on the JS files, but just to prevent future churn
```suggestion
    req.remoteAddress = req.headers['x-forwarded-for']?.split(',')[0].trim() || ws._socket.remoteAddress;
```

Reply:
fixed.